### PR TITLE
[tests-only][full-ci] Test for mail notification to group members if a space is shared

### PR DESCRIPTION
**Pull request by ghost**
## Description
This PR adds the API test scenario for mail notification.
* Scenario: group members get an email notification when someone shares a project space with the group

## Related Issue
* Part of: https://github.com/owncloud/ocis/issues/6606

#### test environment:
* locally

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [X] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [X] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/apiNotification/emailNotification.feature [hunk 1]
```diff
@@ -9,6 +9,7 @@ Feature: Email notification
       | username |
       | Alice    |
       | Brian    |
+      | Carol    |
```
Comment (SwikritiT) [inline tests/acceptance/features/apiNotification/emailNotification.feature:12]: As we don't need this user for all the scenarios, it's not suitable to make it in the background. Please make this user in the scenario where it is needed
```suggestion
```

**Review by SagarGi — APPROVED** [review]
LGTM :+1:

**Review by PrajwolAmatya — COMMENTED** [review]
lgtm

File: tests/acceptance/features/apiNotification/emailNotification.feature [hunk 2]
```diff
@@ -40,3 +40,23 @@ Feature: Email notification
 
       Click here to view it: %base_url%/files/shares/with-me
       """
+
+
+  Scenario: user gets an email notification when someone shares space to group
```
Comment (grgprarup) [inline tests/acceptance/features/apiNotification/emailNotification.feature:45]:
```suggestion
  Scenario: group members get an email notification when someone shares a project space with the group
```

File: tests/acceptance/features/apiNotification/emailNotification.feature [hunk 3]
```diff
@@ -40,3 +40,23 @@ Feature: Email notification
...
+      | role      | viewer |
+    Then the HTTP status code should be "200"
+    And user "Brian" should have received the following email from user "Alice" about the share of project space "new-space"
+      """
+      Hello Brian Murphy,
+
+      %displayname% has invited you to join "new-space".
+
+      Click here to view it: %base_url%/f/%space_id%
+      """
```
Comment (grgprarup) [inline tests/acceptance/features/apiNotification/emailNotification.feature:62]: Mail notification check for another member of group can be added.

File: tests/acceptance/features/apiNotification/emailNotification.feature [hunk 4]
```diff
@@ -40,3 +40,23 @@ Feature: Email notification
 
       Click here to view it: %base_url%/files/shares/with-me
       """
+
+
+  Scenario: user gets an email notification when someone shares space to group
+    Given the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
+    And group "group1" has been created
+    And user "Brian" has been added to group "group1"
```
Comment (grgprarup) [inline tests/acceptance/features/apiNotification/emailNotification.feature:48]: One more user can be added to the group. e.g. Carol

**Review by saw-jan — APPROVED** [review]
LGTM

**Review by grgprarup — APPROVED** [review]
:+1:

**Pull request merged**